configs/prophecy.toml

```diff
@@ -1,14 +1,22 @@
 [workflow]
 input_datasets = ['prophecy']
 sequencing_type = 'genome'
+access_level= 'test'
```

Comment:
I think I actually added this, but we don't need it if running via analysis-runner :) 
```suggestion
```

Reply:
oh neat, so this is redundant? i.e., the analysis runner will automatically know the access level and run the large cohort pipeline accordingly?

Reply:
Yes :)

Reply:
perfect, removing then